app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager.rb

```diff
@@ -22,6 +22,12 @@ class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager < ManageI
           :inverse_of  => :parent_manager,
           :dependent   => :destroy
 
+  has_one :import_auth,
```

Comment:
recommend `has_many` here, even if at first you only want to enable one-at-a-time in the future if you want to make that configurable you'll have to change this

Reply:
changed in code, now testing